templates/firefox-ios.xliff

```diff
@@ -2713,7 +2536,62 @@
         <source>$(PRODUCT_NAME)</source>
       </trans-unit>
       <trans-unit id="CFBundleShortVersionString">
-        <source>14.0</source>
+        <source>15.0</source>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="Shared/Supporting Files/Intro.strings" source-language="en" datatype="plaintext">
```

Comment:
There seem to be a lot of strings here that have already been translated under the Client/Intro.strings file you are deleting. This means that we're losing localizations that already exist, I believe (see any localized file to notice that).
Is there a way to keep the "old" file name, and add new strings there - so that we don't lose any existing localizations?

Reply:
None of the intro strings have actually changed. Just moved. We also had a lot of old Intro strings that we never used so I went ahead and removed them. 
I think these old intro strings were added preemptively back in version 3 or something and have hung around since. I don't know exactly why they are still around in the context of l10n but we dont use them inside the app.

Reply:
That's not how it works. 

The update scripts store translations using file+source string id+hash of the source string (to guard against changes without new IDs).
https://github.com/mozilla-mobile/ios-l10n-scripts/blob/master/update-xliff.py#L124

if you change a filename, you drop all translations from all locales, you're not moving them.

If you want to avoid that, you need to hack together a script (or temporarily change the update one) to remap those strings to the new file. I would suggest to coordinate with Stefan, given that he had to deal in the past to this kind of issues.